packages/emails/src/components/LinkIcon.tsx

```diff
@@ -1,9 +1,11 @@
 import { BASE_URL, IS_PRODUCTION } from "@calcom/lib/constants";
 
-export const LinkIcon = () => (
+export const LinkIcon = ({ secondary }: { secondary?: boolean }) => (
   <img
     src={IS_PRODUCTION ? BASE_URL + "/emails/linkIcon.png" : "https://app.cal.com/emails/linkIcon.png"}
-    width="12px"
+    srcSet={IS_PRODUCTION ? BASE_URL + "/emails/linkIcon.svg" : "https://app.cal.com/emails/linkIcon.svg"}
```

Comment:
Added the possibility to load an SVG to be a crisper image, otherwise it loads usual png